stubs/resources/views/auth/register.blade.php

```diff
@@ -32,12 +32,12 @@
             @if (Laravel\Jetstream\Jetstream::hasTermsAndPrivacyPolicyFeature())
                 <div class="mt-4">
                     <x-jet-label for="terms">
-                        <input class="mr-2 leading-tight" type="checkbox" name="terms" id="terms">
+                        <input class="form-checkbox h-4 w-4 mr-2 text-gray-800 transition duration-150 ease-in-out" type="checkbox" name="terms" id="terms">
 
-                        <span class="mt-1">
+                        <span>
                             {!! __('I agree to the :terms_of_service and :privacy_policy', [
-                                    'terms_of_service' => '<a target="_blank" href="/terms" class="underline text-sm text-gray-600 hover:text-gray-900">'.__('Terms of Service').'</a>',
-                                    'privacy_policy' => '<a target="_blank" href="/policy" class="underline text-sm text-gray-600 hover:text-gray-900">'.__('Privacy Policy').'</a>',
+                                    'terms_of_service' => '<a target="_blank" href="/terms-of-service" class="underline text-sm text-gray-600 hover:text-gray-900">'.__('Terms of Service').'</a>',
```

Comment:
@drjdr You can use `route()` here as well

Reply:
done